build/package/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.redhat.io/rhel9/nodejs-16:1-110 AS builder
+FROM registry.redhat.io/rhel9/nodejs-18:1-108.1714669798 AS builder
```

Comment:
I suggest to add the reference to the container image catalog; I forgot to do that in the first time and I think it saves time when checking for new images.

```suggestion
# https://catalog.redhat.com/software/containers/ubi9/nodejs-18/62e8e7ed22d1d3c2dfe2ca01
FROM registry.redhat.io/rhel9/nodejs-18:1-108.1714669798 AS builder
```

Reply:
For the caddy-ubi container it could be found at: 

```
# https://quay.io/repository/cloudservices/caddy-ubi?tab=tags
```

Reply:
Added the changes.